### ISSUE TITLE
Error on https://e621.net/posts/1200447

### ISSUE THREAD

**Issue by HOZZZMAN** [ruffle-rs/ruffle#7356]
# Error Info
Error name: CompileError
Error message: call to WebAssembly.instantiateStreaming() blocked by CSP
Error stack:
```
@moz-extension://149f47bf-9b16-497c-afda-4e36992caf3a/dist/ruffle.js?uniqueMessageSuffix=25078060288:1:33310
n@moz-extension://149f47bf-9b16-497c-afda-4e36992caf3a/dist/ruffle.js?uniqueMessageSuffix=25078060288:1:33800

```

# Player Info
Allows script access: false

# Page Info
Page URL: https://e621.net/posts/1200447?q=azurebolt

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:102.0) Gecko/20100101 Firefox/102.0
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2021-05-04
Channel: nightly
Built: 2021-05-04T00:13:08.325Z
Commit: f9769451d68e11ec868806b99ddabf14a16bac4e

**Comment by n0samu** [comment]
Duplicate of #2201
